cxf/soap-client-secured/README.md

```
@@ -0,0 +1,144 @@
+curl -sk https://localhost:8443/ws/hello?wsdl
+```
+
+### 2. Run the secured client
+
+```bash
+camel run *
+```
+
+You should see:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)